stackslib/src/chainstate/stacks/db/transactions.rs

```diff
@@ -8544,9 +8546,15 @@ pub mod test {
             fn get_v2_unlock_height(&self) -> u32 {
                 u32::MAX
             }
+            fn get_v3_unlock_height(&self) -> u32 {
```

Comment:
Similar comment as before. Not sure why these are separated by version and pox since they are all identical

Reply:
They're not identical on mainnet.  This trait impl is just for testing transaction-handling.